src/libs/actions/Report.js

```diff
@@ -661,43 +660,6 @@ function fetchInitialActions(reportID) {
  * @param {Boolean} shouldRecordHomePageTiming whether or not performance timing should be measured
  * @returns {Promise}
  */
-function fetchAllReports(
-    shouldRecordHomePageTiming = false,
-) {
-    Onyx.set(ONYXKEYS.IS_LOADING_REPORT_DATA, true);
-    return DeprecatedAPI.Get({
-        returnValueList: 'chatList',
-    })
-        .then((response) => {
-            if (response.jsonCode !== 200) {
-                return;
-            }
-
-            // The cast here is necessary as Get rvl='chatList' may return an int or Array
-            const reportIDs = _.filter(String(response.chatList).split(','), _.identity);
-
-            // Get all the chat reports if they have any, otherwise create one with concierge
-            if (reportIDs.length > 0) {
-                return fetchChatReportsByIDs(reportIDs);
-            }
-
-            return fetchOrCreateChatReport([currentUserEmail, CONST.EMAIL.CONCIERGE], false);
-        })
-        .then((returnedReports) => {
-            Onyx.set(ONYXKEYS.IS_LOADING_REPORT_DATA, false);
-
-            // If at this point the user still doesn't have a Concierge report, create it for them.
-            // This means they were a participant in reports before their account was created (e.g. default rooms)
-            const hasConciergeChat = _.some(returnedReports, report => ReportUtils.isConciergeChatReport(report));
-            if (!hasConciergeChat) {
-                fetchOrCreateChatReport([currentUserEmail, CONST.EMAIL.CONCIERGE], false);
-            }
-
-            if (shouldRecordHomePageTiming) {
-                Timing.end(CONST.TIMING.HOMEPAGE_REPORTS_LOADED);
-            }
```

Comment:
We're leaving a dangling timer here now that will never get the `end()` called 

https://github.com/Expensify/App/blob/ddfc956f4a6ae113fc85f2f22d023c26cdb52eaf/src/libs/Navigation/AppNavigator/AuthScreens.js#L100

I'm not sure if there's a clear way to preserve this behavior and think this metric matters less now that all of the front end logic we were timing is moved to the backend. We can probably just 🔪  this particular trace.